app/controllers/openid_connect/discovery_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OpenidConnect::DiscoveryController < ApplicationController
+  def webfinger
+    jrd = {
+             links: [{
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.